pkg/ovs/openflow/interfaces.go

```diff
@@ -66,6 +66,8 @@ type Bridge interface {
 	DumpFlows(cookieID, cookieMask uint64) map[uint64]*FlowStates
 	// DeleteFlowsByCookie removes Openflow entries from OFSwitch. The removed Openflow entries use the specific CookieID.
 	DeleteFlowsByCookie(cookieID, cookieMask uint64) error
+	// AddFlows adds multiple Openflow entries in a single transaction.
+	AddFlows(flows []Flow) error
```

Comment:
Should we name it AddFlowBundle() to be more clear?

Reply:
Sure, I will update as `AddFlowsInBundle`.